db/re/item_db_usable.yml

```diff
@@ -47553,6 +47595,493 @@ Body:
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,25;
+      getitem 23496,1;
+      getitem 23504,9;
+  - Id: 23496
+    AegisName: Firstaid_Box_65
+    Name: First aid Box (65)
+    Type: Usable
+    EquipLevelMin: 65
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,30;
+      getitem 23497,1;
+      getitem 23505,5;
+  - Id: 23497
+    AegisName: Firstaid_Box_70
+    Name: First aid Box (70)
+    Type: Usable
+    EquipLevelMin: 70
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,30;
+      getitem 23498,1;
+      getitem 23505,6;
+  - Id: 23498
+    AegisName: Firstaid_Box_75
+    Name: First aid Box (75)
+    Type: Usable
+    EquipLevelMin: 75
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,35;
+      getitem 23499,1;
+      getitem 23505,7;
+  - Id: 23499
+    AegisName: Firstaid_Box_80
+    Name: First aid Box (80)
+    Type: Usable
+    EquipLevelMin: 80
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,35;
+      getitem 23500,1;
+      getitem 23505,8;
+  - Id: 23500
+    AegisName: Firstaid_Box_85
+    Name: First aid Box (85)
+    Type: Usable
+    EquipLevelMin: 85
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,40;
+      getitem 22543,2;
+      getitem 23501,1;
+      getitem 23506,7;
+  - Id: 23501
+    AegisName: Firstaid_Box_90
+    Name: First aid Box (90)
+    Type: Usable
+    EquipLevelMin: 90
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,40;
+      getitem 22543,2;
+      getitem 23502,1;
+      getitem 23506,8;
+  - Id: 23502
+    AegisName: Firstaid_Box_95
+    Name: First aid Box (95)
+    Type: Usable
+    EquipLevelMin: 95
+    Weight: 0
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,45;
+      getitem 22543,2;
+      getitem 23506,8;
+  - Id: 23575
```

Comment:
For Id order I mean you put the item ID in wrong order

23502
23575
...
23581
23505 (already in db)

Reply:
oh, thanks, i`ll update it